CHANGELOG.md

```diff
@@ -1,10 +1,64 @@
-## vUNRELEASED
+## v1.4.0
 
 #### New features and UX changes
 
-- Add `--user`/`--group` option to rkt run/prepare ([#2419](https://github.com/coreos/rkt/pull/2419)). This option allows overriding the user/group specified in the image manifest.
+- config: add config subcommand ([#2405](https://github.com/coreos/rkt/pull/2405)). This new subcommand prints the current rkt configuration. It can be used to get i.e. authentication credentials.
```

Comment:
Link to the documentation:
https://github.com/coreos/rkt/blob/master/Documentation/subcommands/config.md